tutorials/physics/interpolation/using_physics_interpolation.rst

```diff
@@ -0,0 +1,154 @@
+input and AI.
+
+Setting the transform of objects only within physics ticks allows the automatic
+interpolation to deal with transforms *between* physics ticks, and ensures the game
+will run the same whatever machine it is run on. As a bonus, this also reduces CPU
+usage if the game is rendering at high FPS, since AI logic (for example) will no
+longer run on every rendered frame.
+
+.. note:: If you attempt to set the transform of interpolated objects *outside* the
+          physics tick, the calculations for the interpolated position will be
+          incorrect, and you will get jitter. This jitter may not be visible on
+          your machine, but it *will* occur for some players. For this reason,
+          setting the transform of interpolated objects should be avoided outside
+          of the physics tick. Godot will attempt to produce warnings in the editor
+          if this case is detected.
+
+.. tip:: This is only a *soft-rule*. There are some occasions where you might want
+         to teleport objects outside of the physics tick (for instance when
+         starting a level, or respawning objects). Still, in general, you should be
+         applying transforms from the physics tick.
+
+
+Ensure that all indirect movement happens during physics ticks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Consider that in Godot, Nodes can be moved not just directly in your own scripts,
+but also by automatic methods such as tweening, animation, and navigation. All
+these methods should also have their timing set to operate on the physics tick
+rather than each frame ("idle"), **if** you are using them to move objects (*these
+methods can also be used to control properties that are not interpolated*).
+
+.. note:: Also consider that nodes can be moved not just by moving themselves, but
+          also by moving parent nodes in the :ref:`SceneTree<class_SceneTree>`. The
+          movement of parents should therefore also only occur during physics ticks.
+
+Choose a physics tick rate
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When using physics interpolation, the rendering is decoupled from physics, and you
+can choose any value that makes sense for your game. You are no longer limited to
+values that are multiples of the user's monitor refresh rate (for stutter-free
+gameplay if the target FPS is reached).
+
+As a rough guide:
+
+.. csv-table::
+    :header: "Low tick rates (10-30)", "Medium tick rates (30-60)", "High tick rates (60+)"
+    :widths: 20, 20, 20
+    
+    "Better CPU performance","Good physics behavior in complex scenes","Good with fast physics"
+    "Add some delay to input","Good for first person games","Good for racing games"
+    "Simple physics behaviour"
+
+.. note:: You can always change the tick rate as you develop, it is as simple as
+          changing the project setting.
+
+Call reset_physics_interpolation() when teleporting objects
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
Call ``reset_physics_interpolation()`` when teleporting objects
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
We use code style within headers, but there isnt much precedent for it and rewording the header might be better overall.

Reply:
not sure what a better way to word it would be so I'll use code style for now.

Reply:
"Resetting physics interpolation when teleporting objects" would work too